cmd/mdatagen/metadata-sample.yaml

```diff
@@ -85,7 +85,7 @@ metrics:
   optional.metric:
     enabled: false
     description: "[DEPRECATED] Gauge double metric disabled by default."
-    unit: 1
+    unit: ""
```

Comment:
Can we add a separate test instead to keep testing this?

Reply:
I want to test `1` specifically, since YAML will interpret this as an integer at first, so it is not exactly equivalent to testing `unit: s`